vignettes/ipums.Rmd

```diff
@@ -1,282 +1,217 @@
 ---
-title: "Introduction to `ipumsr` - IPUMS Data in R"
+title: "IPUMS Data in R"
 author: "Minnesota Population Center"
```

Comment:
All these references to MPC should probably be changed to "Institute for Social Research and Data Innovation".